pwndbg/commands/heap.py

```diff
@@ -80,41 +87,59 @@ def bins(addr=None):
     fastbins = main_arena['fastbinsY']
     bins = main_arena['bins']
 
+    size_t_size = gdb.lookup_type('size_t').sizeof
```

Comment:
Ah, I see that we're skipping over the `prevsize` and `size` field (size_t sized), I thought we were calculating an index into the `fastbinsY` array (pointer-sized).